docs/docs-content/integrations/longhorn.md

```diff
@@ -191,6 +196,76 @@ Longhorn documentation. The examples section contains several use cases of how t
 </TabItem>
 </Tabs>
 
+## Troubleshooting
+
+### Longhorn Deployer Stuck in Init State in Airgap Environments
```

Comment:
Hmm... @lennessyy I'm not sure this needs to be a troubleshooting section. This feels more like configuration guidance. If we were troubleshooting then I would expect steps to debug the issue versus going straight into the configuration, you know? I think we can just make this a subsection of the Usage section.